dspace/config/crosswalks/oai/xoai.xml

```diff
@@ -526,7 +526,7 @@
         </Set>
         <Set id="openaireSet">
             <Spec>openaire</Spec>
-            <Name>OpenAIRE</Name>
+            <Name>Openaire</Name>
```

Comment:
Not sure why this is replaced

Reply:
@frabacche : I do agree with @paulo-graca that the bulk "find & replace" of everything "OpenAIRE" to "Openaire" was not exactly elegant.  It resulted in changing a ton of unrelated code...and, as Paulo notes it modifies some of the human-readable "OpenAIRE" text that is actually visible in OAI-PMH....this line is one example.   

I never intended for a bulk find & replace, but I do realize that's the easiest approach.  I was just wanting to have consistency in our Java class names...and most existing classes (in 7.x) used "OpenAIRE" (camel case), which is why I had recommended that approach.  I do understand why you chose to go with "Openaire" (as it better follows Java naming conventions).  But, it probably would be best to revert the change on this "xoai.xml" line, and double check that there are no other similar changes which are visible to users.

In other words, the User Interface & OAI-PMH should always refer to it as "OpenAIRE" and never "Openaire".  But, I don't mind which version the Java classes themselves use.... we just want the Java naming convention to be _consistent._

Reply:
In all honesty, we can also clean these up again as necessary in future PRs if we discover another place where "Openaire" is accidentally visible to users.  But, I wanted to point out that I agree that users of DSpace should always see "OpenAIRE" as that's the name of the project.  How the Java classes are named doesn't need to match that, but it should be consistent.